src/main/views/macro/form.njk

```diff
@@ -127,9 +127,9 @@
   </div>
 {% endmacro %}
 
-{% macro submitButton(label = 'Continue', id = '') %}
+{% macro submitButton(label = 'Continue', id = '', aria = '') %}
```

Comment:
Can we add ariaLabel instead of aria (aria is more generic)

Reply:
Updated